documentation/reference/consent_decision_making.md

```diff
@@ -0,0 +1,574 @@
+merely a guideline that we expect contributors to adhere to. It cannot cover
+every situation and undoubtedly there are exceptional situations where following
+the process to the letter would be antithetical to the goals of the project and
+even of the process itself.
+
+If you have ideas for improving the Openverse decision-making process, please
+share them. Any contributor may provide feedback on this process.
+
+## Terms
+
+- "Author": The person making the proposal.
+- "Participants": All participants of the decision-making process, including the
+  author, the requested participants, and ad-hoc participants.
+- "Reactions": A broad class of feedback that can include personal preferences
+  as well as general doubts about the validity or safety of the solutions
+  described in the proposal.
+- "Question": A request for clarification, explanation, or additional
+  information about an aspect of the proposal.
+- "Objection": An explained uncertainty about the appropriateness, efficacy, or
+  safety of a solution
+- "Paramount objection": An objection outlining an issue that would cause harm
+  to the project or its contributors; especially issues that could cause harm
+  and are difficult to reverse and not strictly necessary to accomplish the
+  goals of the project
+- "Approved proposal": A proposal that has undergone the formal decision-making
+  process and has been accepted as a valid solution by the participants of the
+  discussion.
+- "Process proposal": A proposal to change processes followed by the
+  contributors in the course of stewarding the project towards success.
+- "Purpose": The motivation for the proposal. This is typically a statement
+  describing the problem the proposal aims to solve. In the original Sociocratic
+  model this is called the "driver".
+
+## Process summary
+
+The process breaks proposal evaluation and decision-making into a series of
+rounds. Each round has expectations about what kinds of feedback are
+appropriate. These steps are ordered and must be followed in the order they are
+presented here.
+
+Full descriptions for each of these rounds are below. This section is a summary
+meant to give an overview for people new to or refreshing themselves on the
+process.
+
+> **Note** The "goal" column is a very brief summary and is not intended to
+> encapsulate the full scope of the round. Please refer to the
+> [round descriptions below](#round-descriptions) for more details and examples.
+
+| round                                                 | Suggested span      | Goal                                                                                      |
+| ----------------------------------------------------- | ------------------- | ----------------------------------------------------------------------------------------- |
+| [Question round](#question-round)                     | 3 days              | Clarify the proposal; share initial reactions                                             |
```

Comment:
What should the process be if there is no response from the maintainers at the end of those 3 (or 5) days? Should we consider that there are no questions, or should we keep the discussion on the current stage until we receive some kind of reply (even if it's "I have no questions") from the assigned reviewers?